src/main/java/uk/gov/hmcts/reform/civil/model/genapplication/GeneralApplication.java

```diff
@@ -25,7 +25,6 @@ public class GeneralApplication implements MappableObject {
     private final GAApplicationType generalAppType;
     private final GARespondentOrderAgreement generalAppRespondentAgreement;
     private final BusinessProcess businessProcess;
-    private final String respondentSolicitor1EmailAddress;
```

Comment:
please check civil service initiategeneralapplicationservicehelper.java line number 100 as i feel we got reference to similar variable name which may require removal  ..and also that might be the reason why its not populating